spec/unit/parser/line_parser_spec.rb

```diff
@@ -14,6 +14,26 @@
     })
   end
 
+  context "unknown entry" do
+    let(:line) { "#MOMSKOD 2611 10"}
+
+    context "lenient parser" do
+      let(:parser) { Sie::Parser::LineParser.new(line, lenient: true) }
+
+      it "raises no error when encountering unknown entries" do
+        expect { parser.parse }.not_to raise_error
+      end
+    end
+
+    context "rigorous parser" do
+      let(:parser) { Sie::Parser::LineParser.new(line) }
+
+      it "raises error when encountering unknown entries" do
+        expect { parser.parse }.to raise_error
+      end
+    end
+  end
```

Comment:
If it had been only up to me, I'd keep the integration test anyway. It still covers the communication between Parser and LineParser.